jamie-sgro/url-shortener
# Feature/#1 create docker environment

## Proposed Changes
  - closes #1
  - Added docker containers and images, complete with scripts to create and run
  - Setup poetry environment with vscode extensions automatically installed with interpreter selected

## Suggested Review Steps

 - Run through readme steps and ensure everything makes sense
 - Ensure test greenlights from container (exit code 0)

## Checklist

- [x] I've added documentation where possible
- [x] I've added unit tests where possible
